portal/questionnaire_template.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * Questionnaire Template
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2022 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once(__DIR__ . "/../src/Common/Forms/CoreFormToPortalUtility.php");
+
+use OpenEMR\Common\Forms\CoreFormToPortalUtility;
+
+// block of code to securely support use by the patient portal
+$patientPortalSession = CoreFormToPortalUtility::isPatientPortalSession($_GET);
```

Comment:
![2DV](https://user-images.githubusercontent.com/278968/181866906-5c179ad4-3579-45ef-946d-df4430f991a1.gif)

Reply:
I really like this Brady. good job